packages/assets/path-support.js

```diff
@@ -4,89 +4,10 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @format
- * @flow strict
+ * @format strict
+ * @flow
```

Comment:
Looks like this line was accidentally rearranged first!

Also, feel free to use `strict-local` everywhere, it's slightly more forgiving.

```suggestion
 * @flow strict-local
 * @format
```